core/src/main/java/org/sonarsource/sonarlint/core/client/api/connected/ConnectedSonarLintEngine.java

```diff
@@ -73,11 +73,22 @@ public interface ConnectedSonarLintEngine extends SonarLintEngine {
    * Gets locally stored server taint issues for a given file.
    *
    * @param projectBinding information about the project (must have been previously updated with {@link #updateProject(EndpointParams, HttpClient, String, boolean, String, ClientProgressMonitor)})
+   * @param branchName     branch name
    * @param filePath       relative to the project.
    * @return All server taint issues in the local storage for the given file. If file has no issues, an empty list is returned.
    */
   List<ServerTaintIssue> getServerTaintIssues(ProjectBinding projectBinding, String branchName, String filePath);
 
+  /**
+   * Gets locally stored server taint issues.
+   *
+   * @param projectBinding information about the project (must have been previously updated with {@link #updateProject(EndpointParams, HttpClient, String, boolean, String, ClientProgressMonitor)})
+   * @param branchName     branch name
+   * @param serverPathToIdePath a function to map a server path to an actual file known by the IDE, if any
+   * @return All server taint issues in the local storage for the given file. If file has no issues, an empty list is returned.
+   */
```

Comment:
I think this annotation needs to be updated? `serverPathToIdePath` is an extra `@param`, and the description - If I understood correctly this method gets all server taint issues for a branch (not the given _file_).